client/src/components/DatabaseConnection.jsx

```diff
@@ -1,34 +1,53 @@
-import React from 'react';
+import React from "react";
 import { useDispatch, useSelector } from "react-redux";
-import { disconnectDB } from '../actions/dbActions';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faLock, faLockOpen } from "@fortawesome/free-solid-svg-icons";
```

Comment:
I'm not the biggest fan of the "lock" and "unlock" symbols to stand for connected/disconnected. It's not very intuitive imo. I preferred the button, although I do agree with the idea of making it stand out a big less.

Reply:
Yeah I didn't think it was the perfect icon either. I just thought it was too many words up there for one feature, so I was trying to find a way to make it take up less space. Let me know if you have any alternatives in mind

Reply:
Should we just have it hidden in a page linked in the navbar? Like an entry for "DB Connection" and when they click it they can view the connection information. Just cause the connection should only need to happen once during setup so I'm not sure if we even need to have it in the header

Reply:
Hmm, that could be a solution, but I kind of like having it on top, since whether or not you're connected is information that could help you easily figure out why you can't add metrics or aren't getting the newest data